### ISSUE TITLE
error: argument name is invalid

### ISSUE THREAD

**Issue by dottodot** [chovy/humanparser#32]
Since v1.8.0 there is a bug that completely breaks my feathersjs app. Just adding the following anywhere makes all request to the app throw the error.

```
const human = require('humanparser');
```

```
error: argument name is invalid
```

**Comment by chovy** [comment]
what version of node are you using?

**Comment by dottodot** [comment]
Sorry v8.11.3

**Comment by chovy** [comment]
It works fine here, can you run `npm test`

**Comment by dottodot** [comment]
npm test did find some issues but nothing that fixed this.

The app will start without an error but as soon as I make a request to any of the endpoints the error is thrown. 

I only have to add `const human = require('humanparser');` to my app.js for this to happen.

What I did find though is that is I go back to 1.7.0 and include the following from the latest version I get the same error

```
Array.prototype.diff = function(a2) {
	return this.concat(a2).filter((val, index, arr) => {
		return arr.indexOf(val) === arr.lastIndexOf(val);
	});
};
```
I also can't replicate on a clean install of feathers so there must be some sort of conflict with another package I have installed which I assume is because `Array.prototype.diff` becomes global so any another package that calls `.diff` on an array will use this. I've done some looking into it and something like the following would work and prevent any conflicts.

```
Object.defineProperty(Array.prototype, 'diff', {
	enumerable: false,
	value: function(a2) {
		return this.concat(a2).filter((val, index, arr) => {
			return arr.indexOf(val) === arr.lastIndexOf(val);
		});
	}
});
```
